src/components/prueba.js

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import React, {useEffect, useState} from 'react';
+
+const Prueba = () => {
+const [token, setToken] = useState([]);
+const [res, setRes] = useState([]);
+
+const apiToken = () => {
```

Comment:
This function should have a name similar to the endpoint. Example: _authenticateUser_ or just _authenticate_.